plasma_framework/contracts/src/framework/utils/Quarantine.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity ^0.5.0;
+
+import "../utils/Operated.sol";
+
+/**
+ * @title Provides a way to quarantine (disable) contracts for a period of time
+ */
+contract Quarantine is Operated {
+    mapping(address => uint256) private _quarantine;
+    uint256 private _quarantinePeriod;
+    uint256 private _immunitiesRemaining;
+
+    /**
+     * @dev The _initialImmuneCount parameter allows us to deploy the platform with some
+     * pre-verified contracts that don't get quarantined.
+     * @param _period the period of time in seconds to quarantine contracts.
+     * @param _initialImmuneCount the first _initialImmuneCount contracts don't get quarantined.
```

Comment:
how about sth like `@param _initialImmuneCount num of contracts that don't get quarantined at first.` ?